Async-Simulator/README.md

```diff
@@ -0,0 +1,14 @@
+#AsyncApi Performance Tester
+___
+####UNDER DEVELOPMENT
+
+Npm library that aims to make test your backend services at scale 
```

Comment:
Npm library which aim is to making your backend services testable at scale. @NektariosFifes what do you think?